package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "apidocs": "./node_modules/.bin/jsdoc2md",
     "browser-tests": "NODE_ENV=test ./node_modules/.bin/karma start --browsers Chrome",
-    "build": "NODE_ENV=production npm run clean && ./node_modules/.bin/webpack --config package.config.js",
+    "build": "NODE_ENV=production npm run clean && NODE_ENV=production ./node_modules/.bin/webpack --config package.config.js",
```

Comment:
Do you need to reset NODE_ENV twice? Or is it a typo?

Reply:
> Do you need to reset NODE_ENV twice? Or is it a typo?

We need it for the travis build